src/Core/src/Platform/iOS/CharacterSpacingExtensions.cs

```diff
@@ -0,0 +1,58 @@
+﻿using Foundation;
+using UIKit;
+
+namespace Microsoft.Maui
+{
+	public static class CharacterSpacingExtensions
+	{
+		public static NSMutableAttributedString? AddCharacterSpacing(this NSAttributedString attributedString, string text, double characterSpacing)
+		{
+			if (attributedString == null && characterSpacing == 0)
+				return null;
+
+			NSMutableAttributedString? mutableAttributedString = attributedString as NSMutableAttributedString;
+
+			if (attributedString == null || attributedString.Length == 0)
```

Comment:
Should this be checking `mutableAttributedString ` instead? If not, then what is the logic above for? We could just have a variable definition. But this check also does not seem to fit.